rpc/doc.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Overview
+// Package rpc provides the underpinnings of requests and responses (server.Request)
+// and dispatching them to the API by defining a request
+// rpc provides the idea of a request that has a named type, a version, and a method to be called
```

Comment:
```suggestion
// More specifically, the package provides the idea of a request that has a named type, a version, and a method to be called
```

Reply:
removed the overview to combine with sequence flow